### ISSUE TITLE
'inspect raw --basedir esc-10' is not an audeep command.

### ISSUE THREAD

**Issue by ben-williams-ai** [auDeep/auDeep#42]
**OS Platform and Distribution: Windows 10
** auDeep version: 0.9.4
**Python version: 3.7

Hi there, I followed the instructions carefully in a new anaconda environment and everythig installed correctly. However, when I enter the fifth line in the example walkthrough:

audeep inspect raw --basedir ESC-10

I get an error:

audeep: 'inspect raw --basedir esc-10' is not an audeep command. See 'Audeep --help'. 
Did you mean of these?
   Complete

I've also tried this with the ESC-50 dataset. Any suggestions as to how I can fix this would be appreciated.

**Comment by mauricege** [comment]
Hi!

Which version of auDeep are you using? Did you install from pip? It may be an issue with cliff: Try downgrading it to a version such that `cliff>=3.3,<3.4` or upgrade auDeep to the newest pre-release version (0.9.6a1).

**Comment by ben-williams-ai** [comment]
Hi, thanks for the quick reply and useful help. I used `pip install cliff == 3.3` to install an older version of cliff.

This worked but a new error came up: AttributeError: module 'pandas' has no attribute 'Panel'. So, I installed an older version of pandas using `pip install pandas==1.0` which I can see is the minium req for auDeep. This is now working and printed the table. Sharing the details in case anyone else encounters the same issues.

Also installing the latest version using `pip install audeep==0.9.6a1` fixed this
